packages/fxa-auth-server/lib/senders/emails/templates/downloadSubscription/index.stories.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/downloadSubscription',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'downloadSubscription',
+  'Sent when a user adds a subscription.',
```

Comment:
Small nit: we could use the word "successfully" since if their payment fails they're sent something else.